### ISSUE TITLE
Mega Suggestion Google Doc

### ISSUE THREAD

**Issue by RaliaTheWolf** [capnkirok/animania#243]
First off, I must warn you, there is A TON of stuff on here. Like, seriously, I went WAAAY overboard. But I couldn't help myself... I love animals and this mod so much and I got to thinking and this thing happened... 
I understand that there is absolutely no way that you could possibly add all of this stuff, but I would love to help out with this mod in any way I can and this was just a way I thought I could help... I tried to organize it (formatting is my middle name) and add some of my weird sense of humor to make this ridiculously long doc somewhat enjoyable to scroll through... I'll go through it and cross out stuff that has been added or is definitely not being added to keep it tidy! I also might add some more stuff, although there is definitely enough there already... If there's anything you don't understand, let me know!
Long story short, I made a thing; feel free to browse through it and comment and stuff!! Thank you for your time and for making us this amazing mod! ^^

Link: https://docs.google.com/document/d/1CfkU9AsWqZiqXO2mhvoX4AHIa7ey7mTjW_JaA9cY-OU/edit?usp=sharing

PS: I apologize if I should have put this somewhere else... I can move it if you need me to!

**Comment by Tschipp** [comment]
Thanks, many of these are already planned, you also left some other quality suggestions. We will consider it

**Comment by RaliaTheWolf** [comment]
Thank you! I'm glad you liked them!

**Comment by capnkirok** [comment]
These are great Ralia, I love your excitement :)

**Comment by RaliaTheWolf** [comment]
Thank you! ^^

**Comment by capnkirok** [comment]
Hey @RaliaTheWolf I've added your whole doc to our offline suggestions list. Some of it is in already as you know. Going to close this one now in an effort so focus our git on bugs only. Thanks for all of the support!
